x-pack/plugins/alerts/server/task_runner/task_runner.ts

```diff
@@ -411,18 +411,20 @@ function generateNewAndResolvedInstanceEvents(params: GenerateNewAndResolvedInst
   }
 
   for (const id of currentAlertInstanceIds) {
-    const message = `${params.alertLabel} active instance: '${id}'`;
-    logInstanceEvent(id, EVENT_LOG_ACTIONS.activeInstance, message);
+    const { actionGroup } = currentAlertInstances[id].getScheduledActionOptions()!;
```

Comment:
I like to avoid having to use `!` in code, I feel like it's going to bite me in the future :-)

Could we change this to something like

```suggestion
    const actionGroup = currentAlertInstances[id].getScheduledActionOptions()?.actionGroup;
```

I think that would make the `message` below end up with `in actionGroup: 'undefined'`, which is likely fine for the actual message in the event log, as that field is purely for use by humans - we don't parse it up or anything.

Reply:
Updated!